src/renderer/components/message/MessageMarkdown.tsx

```diff
@@ -103,10 +103,59 @@ function renderElement(elm: ParsedElement, key?: number): JSX.Element {
   }
 }
 
-export function message2React(message: string): JSX.Element {
+/** render in preview mode for ChatListItem summary, not interactive but looks rendered */
```

Comment:
I'm not sure if I understand this comment, maybe the word order is off? How about something like: "Render formatted summary text of chat list item". The "interactivity" part I don't know if I understand. Maybe you want to say: "Links can not be clicked." etc.?